microchip_SW/README.md

```diff
@@ -0,0 +1,2 @@
+# hlavo-station
```

Comment:
- popsat zde adresářovou strukturu a jak se to zprovozní (IDE, specifika překladu a pod.)
- libraries/hlavo - co tato knihovna obsahuje? Popsat v README, jinak je v libraries předpokládám převzatý kód
- musí se adresář "examples" takto jmenovat?
- jsou všechny libraries využívané? Do budoucna může být trochu problém to udržovat takže pokud se blížíme k "produkční verzi" tak by chtělo zredukovat co se nepoužívá

Reply:
- popsano
- `examples` je prevzato kvuli tomu, aby se to zobrazovalo v ArduinoIDE, pro Platformio je zcela jedno. I v ArduinoIDE to lze asi prestrukturovat, ale nehral jsem si s tim
- ano, jsou, verze jsou popsany, muzeme z toho pak udelat submoduly, jestli nam to stoji za to